openmdao/docs/_utils/run_sub.py

```diff
@@ -0,0 +1,36 @@
+"""
+This is used by our doc build system to execute a code chunk in a subprocess while giving that code chunk
+access to its containing module's globals.
+"""
```

Comment:
Does 'exec' run in a subprocess?

Reply:
It does, but we only do this for embedded code that generates plots.  It's necessary because running a whole bunch of matplotlib stuff all in the same process causes issues with overlapping figures and such.  I tried clearing out the current figure in our modified code in various ways but it never worked right.